{"name": "test_wpa3_personal_mixed_ssid_5g[setup_profiles0]", "status": "broken", "statusDetails": {"message": "IndexError: list index out of range", "trace": "get_configuration = {'access_point': [{'ip': '10.28.3.103', 'jumphost': True, 'jumphost_tty': '/dev/ttyAP5', 'mode': 'wifi6', ...}], 'cont...tation-Name': 'wlan0', '5G-Radio': ['1.1.wiphy1', '1.1.wiphy3'], '5G-Station-Name': 'wlan0', ...}, 'name': 'lanforge'}}\nget_apnos = <class 'apnos.apnos.APNOS'>\nfixtures_ver = <fixtures_2x.Fixtures_2x object at 0x7f7cb587cf40>\nadd_allure_environment_property = <function add_allure_environment_property.<locals>.maker at 0x7f7cb590a040>\n\n    @fixture(scope='session')\n    def add_env_properties(get_configuration, get_apnos, fixtures_ver, add_allure_environment_property: Callable) -> None:\n        add_allure_environment_property('Access-Point-Model', get_configuration[\"access_point\"][0][\"model\"])\n        add_allure_environment_property('Access-Point-Firmware-Version',\n>                                       fixtures_ver.get_ap_version(get_apnos, get_configuration)[0].split(\"\\n\")[1])\nE       IndexError: list index out of range\n\nconftest.py:645: IndexError"}, "description": "Client Connectivity open ssid 2.4G\n           pytest -m \"client_connectivity and vlan and general and wpa3_personal_mixed and fiveg\"\n        ", "parameters": [{"name": "setup_profiles", "value": "{'mode': 'VLAN', 'ssid_modes': {'wpa3_personal': [{'ssid_name': 'ssid_wpa3_p_2g', 'appliedRadios': ['2G'], 'security_key': 'something', 'vlan': 100}, {'ssid_name': 'ssid_wpa3_p_5g', 'appliedRadios': ['5G'], 'security_key': 'something', 'vlan': 100}], 'wpa3_personal_mixed': [{'ssid_name': 'ssid_wpa3_p_m_2g', 'appliedRadios': ['2G'], 'security_key': 'something', 'vlan': 100}, {'ssid_name': 'ssid_wpa3_p_m_5g', 'appliedRadios': ['5G'], 'security_key': 'something', 'vlan': 100}], 'wpa_wpa2_personal_mixed': [{'ssid_name': 'ssid_wpa_wpa2_p_m_2g', 'appliedRadios': ['2G'], 'security_key': 'something', 'vlan': 100}, {'ssid_name': 'ssid_wpa_wpa2_p_m_5g', 'appliedRadios': ['5G'], 'security_key': 'something', 'vlan': 100}]}, 'rf': {}, 'radius': False}"}], "start": 1631664167048, "stop": 1631664167048, "uuid": "e3a9834d-dcd9-4eb2-9b48-737a8ff755b4", "historyId": "a61245fd599dd8d805bddd472facd245", "testCaseId": "01bb566a08d560cd97755a7f85256d69", "fullName": "e2e.basic.validation_of_operating_modes.vlan_mode.client_connectivity.test_general_security_modes.TestvlanModeConnectivitySuiteTwo#test_wpa3_personal_mixed_ssid_5g", "labels": [{"name": "story", "value": "open 5 GHZ Band"}, {"name": "feature", "value": "vlan MODE CLIENT CONNECTIVITY"}, {"name": "tag", "value": "sanity"}, {"name": "tag", "value": "client_connectivity"}, {"name": "tag", "value": "ucentral"}, {"name": "tag", "value": "suiteB"}, {"name": "tag", "value": "fiveg"}, {"name": "tag", "value": "general"}, {"name": "tag", "value": "@pytest.mark.usefixtures('setup_profiles')"}, {"name": "tag", "value": "uc_sanity"}, {"name": "tag", "value": "wpa3_personal_mixed"}, {"name": "tag", "value": "vlan"}, {"name": "parentSuite", "value": "e2e.basic.validation_of_operating_modes.vlan_mode.client_connectivity"}, {"name": "suite", "value": "test_general_security_modes"}, {"name": "subSuite", "value": "TestvlanModeConnectivitySuiteTwo"}, {"name": "host", "value": "testing-139-6tsxh"}, {"name": "thread", "value": "6-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "e2e.basic.validation_of_operating_modes.vlan_mode.client_connectivity.test_general_security_modes"}]}